docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_label: "Epinio Journey: Single Dev Workflow"
+title: ""
+---
+
+## Introduction
+
+While the [quickstart](./quickstart.md) is great to get started, as a Developer, you might want to see end-to-end solutions and how Epinio can help you.
+That's exactly the aim of the "Epinio Journeys", where you'll be able to follow different use-cases, according to your needs.
+
+In this particular tutorial, we focus on the single dev workflow and go through a full example from Epinio installation to your first app deployment.
```

Comment:
This might improve flow: 
```
In this particular tutorial, we focus on the workflow for a solo developer with an example from a bare Kubernetes deployment to your first app.
```

Reply:
It might also be good to call out that this is for local development as well